components/button/button-icon.js

```diff
@@ -174,7 +176,7 @@ class ButtonIcon extends ThemeMixin(ButtonMixin(VisibleOnAncestorMixin(RtlMixin(
 				name="${ifDefined(this.name)}"
 				title="${ifDefined(this.text)}"
 				type="${this._getType()}">
-				<d2l-icon icon="${ifDefined(this.icon)}" class="d2l-button-icon"></d2l-icon>
```

Comment:
Hmm, the `.d2l-button-icon` CSS class is forcing a `0.9rem` width and height -- presumably because consumers can use tier2 and tier3 icons here and that would make the button huge. Maybe we want to do something similar? Like adjust the CSS rule to target `d2l-icon` and `d2l-icon-custom` inside the slotted content?